jjb/ci-management/ci-jobs.yaml

```diff
@@ -5,6 +5,9 @@
 
     project: ci-management
     branch: main
+    stream:
+      - main:
```

Comment:
no need to set branch in the stream (you've got it on the line above) also, no need to do this as an array since you're only doing one stream.